### DISCUSSION TITLE
When you change your icon theme, could it also change in the Qt5 Settings?

### DISCUSSION THREAD

**Discussion by moloch1994** [linuxmint/discussions#382] [Ideas]
With the recent Styles feature letting you switch between accent colors so easily, it almost encourages you to change it to match the wallpaper colors. It would be nice if the icons changed system-wide though, without forcing you to go to Qt settings manually and change it from there as well. I feel like this shouldn't be too difficult to set up, it probably just requires a script.

**Comment by clefebvre** [comment]
I'd prefer not to mess with Qt settings. They tend to change between versions. We tried to overwrite them in the past and it kinda backfired on us.

Unless the Qt team sets a standard and is OK for us to do that of course, but until then I'd prefer not to mess with them.
